### ISSUE TITLE
Error installing on python 3.11 on Windows and running moduleEigengenes

### ISSUE THREAD

**Issue by dmitriikamaevqia** [mortazavilab/PyWGCNA#62]
Hi,
I cannot install the package on windows with python 3.11. There are some problems with installation old pandas version. Is there a reason you pinned pandas version?

Additionally, I tried to unpin pandas version and installed the package from source. Then I encountered an error running Quick Start.ipynb 

I believe in wgcna.py inside moduleEigengenes function
`covEx[not np.isfinite(covEx)] = 0` should instead be
`covEx[~ np.isfinite(covEx)] = 0`

Also to support new pandas version append on dataframe should be replaced with concat around line 1979
```
tmp = pd.concat([tmp, pd.DataFrame(v[0:min(n, p, nVarExplained), :],
                                              columns=tmp.columns.tolist())])
```

utils line 129:
`
geneInfo = pd.concat([geneInfo, pd.DataFrame.from_records([dict])])`

**Comment by nargesr** [comment]
Hi,

I haven’t had a chance to update the package to be compatible with newer pandas. It's on my to-do list but might take a while. 

if you can downgrade your Python version, I think you should be fine with the older Pandas version. Meanwhile, I'll try to update it to new version this month

**Comment by zijunmeng** [comment]
> Hi,
> 
> I haven’t had a chance to update the package to be compatible with newer pandas. It's on my to-do list but might take a while.
> 
> if you can downgrade your Python version, I think you should be fine with the older Pandas version. Meanwhile, I'll try to update it to a new version.

could u update PyWGNA as soon as possible, the installation of pywgcna was excruciating.
Don't submit any more unimportant updates.

**Comment by nargesr** [comment]
Hi,

I just released a new version(2.0.0) which is compatible with Pandas > 2.0.0

Please feel free to reopen this issue if you get any error
